app/components/latest-releases.ts

```diff
@@ -0,0 +1,68 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import type Store from '@ember-data/store';
+import type CourseExtensionIdeaModel from 'codecrafters-frontend/models/course-extension-idea';
+import type CourseIdeaModel from 'codecrafters-frontend/models/course-idea';
+
+interface Signature {
+  Element: HTMLDivElement;
+}
+
+interface Release {
+  timestampText: string;
+  type: 'extension' | 'challenge';
+  title: string;
+  releaseDate: Date;
+  announcementUrl?: string;
+}
+
+export default class LatestReleasesComponent extends Component<Signature> {
+  @service declare store: Store;
+
+  get releases(): Release[] {
+    const courseExtensionIdeas = this.store.peekAll('course-extension-idea').filterBy('developmentStatusIsReleased');
+    const courseIdeas = this.store.peekAll('course-idea').filterBy('developmentStatusIsReleased');
+
+    const releases: Release[] = [];
+
+    courseExtensionIdeas.forEach((idea: CourseExtensionIdeaModel) => {
+      if (idea.releasedAt) {
+        releases.push({
+          timestampText: this.formatTimestamp(idea.releasedAt),
+          type: 'extension',
+          title: idea.name,
+          releaseDate: idea.releasedAt,
+          announcementUrl: idea.announcementUrl || undefined,
+        });
+      }
+    });
+
+    courseIdeas.forEach((idea: CourseIdeaModel) => {
+      if (idea.releasedAt) {
+        releases.push({
+          timestampText: this.formatTimestamp(idea.releasedAt),
```

Comment:
We don't need to compute this here, look into using the `{{date-format` helper. That way we don't have this duplication where we have to keep both `releaseDate` and `timestampText` in the same hash.

Reply:
Also, we only need months - no dates: 

<img width="152" alt="Screenshot 2025-07-09 at 23 03 01" src="https://github.com/user-attachments/assets/6d7110a2-e9f2-474e-b819-549d5256f89c" />